.resources/strings.json

```diff
@@ -72,38 +80,43 @@
 		"ja": "サービス",
 		"es": "Servicios",
 		"de": "Services"
+		"pt-BR": "Serviços"
```

Comment:
This colon character is highlighted in the PR code for some reason can you change it to a normal colon character if it's some weird unicode version?